esp-hal-procmacros/src/lib.rs

```diff
@@ -332,7 +332,7 @@ pub fn handler(args: TokenStream, input: TokenStream) -> TokenStream {
         #f
 
         #[allow(non_upper_case_globals)]
-        #vis static #orig: #root::interrupt::InterruptHandler = #root::interrupt::InterruptHandler::new(#new, #priority);
+        #vis const #orig: #root::interrupt::InterruptHandler = #root::interrupt::InterruptHandler::new(#new, #priority);
```

Comment:
I think this was `static` for a reason, so it gets placed in RAM directly, so we _probably_ don't want this change.

Reply:
What needs to be in RAM, the function item or the constant priority? The pointer is copied into the vector table, and the priority is written to the hardware.

Reply:
Probably the reason for `static` was to place it in RAM but without a link-section and not being `mut` this will go to DROM anyways I guess - not much difference if it ends up as an anonymous symbol or not, then

Any case it's only read when setting the vector table entry and the priority, so I guess this is fine